cartography/intel/gcp/iam.py

```diff
@@ -356,9 +409,12 @@ def load_bindings(neo4j_session: neo4j.Session, bindings: List[Dict], project_id
             if member.startswith('user:'):
                 usr = member[len('user:'):]
                 user = {
-                    'id': f"projects/{project_id}/users/{usr}",
```

Comment:
will this affect when do match on the user object?

Reply:
the `id` field contain email of entity and email id always unique and if any iam entity has access to multiple projects then it will creates multiple entries with different `id` so it will difficult to check the how many projects access have to the entity

Reply:
makes sense.